### PR TITLE
Documenting backup workflows and actions in backups.md

### PR DESCRIPTION
**Pull request by mansueli**
## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: docs/backups.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Supabase Backup Database
+
+This guide will show you how to use the Supabase CLI to backup your Postgres database. The steps will involve running a series of commands to dump roles, schema, and data separately.
```
Comment (kiwicopple) [inline docs/backups.md:3]:
```suggestion
You can use the Supabase CLI to backup your Postgres database. The steps involve running a series of commands to dump roles, schema, and data separately.
```

File: docs/backups.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Supabase Backup Database
```
Comment (kiwicopple) [inline docs/backups.md:1]:
```suggestion
# Backup your database
```

File: docs/backups.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      - name: Backup roles
+        run: supabase db dump --db-url "$supabase_db_url" -f roles.sql --role-only
+      - name: Backup schema
+        run: supabase db dump --db-url "$supabase_db_url" -f schema.sql
+      - name: Backup data
+        run: supabase db dump --db-url "$supabase_db_url" -f data.sql --data-only --use-copy
+```
+## Periodic Backups Workflow
+
+This section describes a workflow that creates periodic backups of your Supabase database. The workflow is triggered by `push` and `pull_request` events on the `main` branch, manually via `workflow_dispatch`, and automatically at midnight every day due to the `schedule` event with a `cron` expression.
```
Comment (kiwicopple) [inline docs/backups.md:31]:
```suggestion
You can use the GitHub Action to run periodic backups of your database. In this example, the Action workflow is triggered by `push` and `pull_request` events on the `main` branch, manually via `workflow_dispatch`, and automatically at midnight every day due to the `schedule` event with a `cron` expression.
```

**Review by kiwicopple — APPROVED** [review]

**Comment by wyozi** [comment]
@kiwicopple @mansueli this is imo simply a bad idea and kind of weird to have in official Supabase docs, as if you were advocating this.

Few reasons:
- Git repos are just not the right place to store backups. In addition, Github has a 100mb limit so this will fail for any reasonable sized database
- Storing customer data in form of backups in a Git repo also is weird. Almost like storing access tokens in .env files
- The example action is just implemented weird. Why would you want backup action to be triggered by this: "the action will run whenever a new PR is created."? Why is it triggered by pushes to the repo as well?

I would suggest either changing this to something more reasonable (like uploading backups to s3 in a github action, which is a bit weird but less weird) or just removing it altogether.

**Pull request merged**